lib/src/test/java/io/ably/lib/test/realtime/ConnectionManagerTest.java

```diff
@@ -603,7 +618,10 @@ public void onChannelStateChanged(ChannelStateChange stateChange) {
 					ably.close();
 				}
 			});
+            (new Helpers.ChannelWaiter(channel)).waitFor(ChannelState.attached);
```

Comment:
Will you please sort out your IDE/editor?  It's getting tiring spending time commenting on whitespace I am afraid.

Reply:
Also this is an old commit and is now fixed.